scripts/aws-clean-up.js

```diff
@@ -0,0 +1,51 @@
+const AWS = require('aws-sdk');
+
+const region = 'us-east-1';
+
+const s3 = new AWS.S3();
+const cf = new AWS.CloudFormation({ region });
+const lambda = new AWS.Lambda({ region });
+
+const prefix = 'ask-';
+
+const deleteBucket = async (name) => {
+    try {
+        const versions = await s3.listObjectVersions({ Bucket: name }).promise();
+
+        let deletePromises = versions.Versions.map(i => s3.deleteObject({ Bucket: name, Key: i.Key, VersionId: i.VersionId }).promise());
+        await Promise.all(deletePromises);
+
+        deletePromises = versions.DeleteMarkers.map(i => s3.deleteObject({ Bucket: name, Key: i.Key, VersionId: i.VersionId }).promise());
+        await Promise.all(deletePromises);
+
+        await s3.deleteBucket({ Bucket: name }).promise();
+    } catch (e) { console.log('Unable to delete ', name); }
```

Comment:
i think the "e" itself is also better to be displayed so that we know the reason